json-server/users-profile.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  'parent1@example.com': {
+    dob: '1976-05-22T00:00:00.000Z',
+  },
+  'parent2@example.com': {
+    dob: '1969-11-26T00:00:00.000Z',
+  },
+  'child1o13@example.com':{
+    dob: '2000-10-26T00:00:00.000Z',
```

Comment:
Tests using this child will break next year 😅

Reply:
true